uid2_client/request_response_util.py

```diff
@@ -20,8 +22,7 @@ def auth_headers(auth_key):
     return {'Authorization': 'Bearer ' + auth_key,
             "X-UID2-Client-Version": "uid2-client-python-" + version}
 
-
-def make_v2_request(secret_key, now, data=None):
+def create_envelope(secret_key, now, data=None) -> 'Envelope':
```

Comment:
It's nice that we started adding typing. Could we add them consistently to function inputs and outputs in this file as well?

Reply:
where is typing being consumed? are we adding mypy or similar in the pipelines?

Reply:
Have you tested with a type checker locally though? Good to check all the typing is correct before merging

Sounds good to add one in pipelines later - do you mind making a ticket?